wherehows-web/app/typings/api/list/compliance-datatypes.d.ts

```diff
@@ -0,0 +1,35 @@
+import { Classification, ComplianceFieldIdValue, IdLogicalType } from 'wherehows-web/constants';
+import { ApiStatus } from 'wherehows-web/utils/api/shared';
+
+/**
+ * Describes the interface for a complianceDataType
+ * @export
+ * @interface IComplianceDataType
+ */
+export interface IComplianceDataType {
+  // Indicates that id contains pii
+  pii: boolean;
+  // Flag indicating if a field with this id is a member identifier
+  idType: boolean;
```

Comment:
idType being a boolean strikes me as a weird way to name the property.

Reply:
Yea, I feel the same way too, but unfortunately, this is a mirror for the api response shape :/ @cptran777